core/jobs.py

```diff
@@ -103,7 +103,13 @@ def _is_abstract(cls):
 
     @classmethod
     def create_new(cls):
-        """Creates a new job of this class type. Returns the id of this job."""
+        """Creates a new job of this class type.
+        Returns:
+            The id of this job.
+        Raises:
+            Exception: A job using an abstract base manager tried to directly
```

Comment:
Exception: This method (instead of a subclass method) was directly used to create a new job.